### ISSUE TITLE
Toolbox could not be loaded in SSMS v18.9.1

### ISSUE THREAD

**Issue by svenroos** [ErikEJ/SqlCeToolbox#898]
Hi,

first thanks to your toolbox.
I have followed the guide and "installed" it for SSMS (i.e. copied it to folder under \Extension).
A In the Tools Menu it shows the "SQLIte / SQL Server Compact Toolbox"; same in View\Other windows
B In Options it has entry "SQLite/SQLCEToolbox".

### Steps to reproduce
A Click on menu item. A error message "Could not load file or assembly 'Microsoft.VisuqlStudio.Shell.Immutable.10.0, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. System cannot find the requested file." [Note: Last sentence is translated. On my computer it is shown in German as I am using German OS]
B Click on entry. Error "An error occured loading this properyt page" is shown

### Further technical details
Toolbox/Power Tools version: vsix Filename is ...v4.7.691 (About dialog - not possible)

Database engine: n.a.

Visual Studio or SSMS version: SSMS v18.9.1 Component SQL Server Management Studio: Version 15.0.18384.0

I haven't found any log file.

**Comment by ErikEJ** [comment]
I think you might need VS Community installed as well...

**Comment by svenroos** [comment]
Thanks.
I do have Visual Studio Community 2019 Version 16.9.1 installed.
Toolbox is working fine in Visual Studio.
But I would require Toolbox in SSMS also.

**Comment by svenroos** [comment]
I found a solution. 
I had to manually add the correct version of Microsoft.VisualStudio.Shell.Immutable.10.0.dll to the GAC. Actually an older version (10.0.30319) than the current one (16.9.31024.71 as by today). 
The one from [NuGet Gallery -  Microsoft.VisualStudio.Shell.Immutable.10.0/10.0.30319](https://www.nuget.org/packages/Microsoft.VisualStudio.Shell.Immutable.10.0/10.0.30319) worked for me:
1. Download package from [NuGet Gallery](https://www.nuget.org/packages/Microsoft.VisualStudio.Shell.Immutable.10.0/10.0.30319)
2. Unzip and extract the Microsoft.VisualStudio.Shell.Immutable.10.0.dll from \lib\net40 and copy to "_\anyplace_"
3. Start VS with administration rights 
4. Add to GAC via VS Terminal and gacutil -if "_\anyplace_\Microsoft.VisualStudio.Shell.Immutable.10.0.dll"

Happy if you add it to WiKi or installation hints etc. Credits welcome but not required.
Thanks for the toolbox!
